uart/lcd_uart/README.adoc

```diff
@@ -0,0 +1,44 @@
+= Attaching a 16x2 LCD via TTL
+
+This example code shows how to interface the Raspberry Pi Pico to one of the very common 16x2 LCD character displays. Due to the large number of pins these displays use, they are commonly used with extra drivers or backpacks. In this example, we will use an Adafruit LCD display backpack, which supports communication over USB or TTL. An RGB display is also used, but the backpack is compatible with white/blue or one color displays too.
```

Comment:
Might be worth mentioning the https://github.com/raspberrypi/pico-examples/tree/master/i2c/lcd_1602_i2c example here? :wink: